src/effects/effectchainslot.cpp

```diff
@@ -68,6 +68,12 @@ EffectChainSlot::EffectChainSlot(EffectRack* pRack, const QString& group,
 
     connect(&m_channelStatusMapper, SIGNAL(mapped(const QString&)),
             this, SLOT(slotChannelStatusChanged(const QString&)));
+
+    // It is up to skins and controller mappings to do anything useful with this.
+    m_pControlExpanded = new ControlPushButton(ConfigKey(m_group, "expanded"));
```

Comment:
IMO if we have skins and and controllers creating COs to communicate it is probably because we have failed at designing an interface that serves their purposes. We already have a mess with controller scripts including user-configurable options to change their behavior to correspond better with different skins. Let's not dig deeper into that hole.

We do need a way for controller mappings to communicate with each other, but that's a separate issue.